clients/drcachesim/common/options.cpp

```diff
@@ -776,5 +777,58 @@ droption_t<bool> op_enable_kernel_tracing(
     "Processor Trace.");
 #endif
 
+// Core-oriented analysis.
+droption_t<bool> op_core_sharded(
+    DROPTION_SCOPE_ALL, "core_sharded", false, "Analyze per-core in parallel.",
+    "By default, the input trace is analyzed in parallel across shards equal to "
```

Comment:
We should also add details of the new features to a page on https://dynamorio.org/page_drcachesim.html.

Reply:
+1, should have done this originally.  I augmented `sec_drcachesim_newtool` to talk about both shard types, how to identify the shard type, and how the dynamic schedule is affected by analysis time and how a replayed schedule can avoid problems there if someone has an asymmetric-overhead tool.  I also added a mention in `sec_drcachesim_sched` (which itself needs to be expanded substantially for the scheduler in general but that will be a separate PR).